<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="startPb">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopPb">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="statusTb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>491</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="ipAddrTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <column>
    <property name="text">
     <string>IP Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MAC Address</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="senderLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sender IP:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="senderIpTb">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="targetIpTb">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="targetLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target IP:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
